projects/jdwp/serializers/serializer_codegen.py

```diff
@@ -0,0 +1,99 @@
+import re
+import os
+from projects.jdwp.defs.schema import Type, Struct, CommandSet, Command, Field
+from projects.jdwp.defs.command_sets.reference_type import ReferenceType
+
+
+class CommandSerializerGenerator:
+    def __init__(self):
+        self.serializer_code: str = ""
+
+    def generate_field_serializer(self, field: Field) -> str:
+        serializer_code: str = ""
+        if isinstance(field, Struct):
```

Comment:
This should never be true, this should be (and I believe is) guaranteed on the type system level.